daemon/archive_tarcopyoptions_unix.go

```diff
@@ -14,12 +18,21 @@ func (daemon *Daemon) tarCopyOptions(container *container.Container, noOverwrite
 		return daemon.defaultTarCopyOptions(noOverwriteDirNonDir), nil
 	}
 
-	user, err := idtools.LookupUser(container.Config.User)
+	user, err := user.Lookup(container.Config.User)
 	if err != nil {
 		return nil, err
 	}
 
-	identity := idtools.Identity{UID: user.Uid, GID: user.Gid}
+	uid, err := strconv.Atoi(user.Uid)
+	if err != nil {
+		// Should always be numeric on POSIX systems.
```

Comment:
It's documented on [the `user.User` type](https://pkg.go.dev/os/user#User) and thus falls under the Go 1 compatibility promise.